packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1671,6 +1674,130 @@ class _AnimatedRotationState extends ImplicitlyAnimatedWidgetState<AnimatedRotat
   }
 }
 
+/// Animated version of [Transform.translate] which automatically transitions the child's
+/// offset relative to its normal position whenever the given offset changes.
```

Comment:
This doesn't seem right. Inside the widget you use `SlideTransition`, which behaves differently from the `Transform.translate`. The paragraph below seems to be on point, but I'm not sure if that's the behaviour you would expect from a widget with name `AnimatedOffset`

Reply:
Maybe then it makes sense to rename to `AnimatedSlide`? This way naming would be consistent with `SlideTransition`, which is actually used under the hood